features/saved_searches/ownerships.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Test the UDB3 ownerships saved searches API
+  
+  Background:
+    Given I am using the UDB3 base URL
+    And I am using an UiTID v1 API key of consumer "uitdatabank"
+    And I am authorized as JWT provider user "centraal_beheerder"
+    And I send and accept "application/json"
+    And I create a minimal organizer and save the "id" as "organizerId"
+    And I am authorized as JWT provider user "invoerder_ownerships"
+    And I request ownership for "auth0|64089494e980aedd96740212" on the organizer with organizerId "%{organizerId}" and save the "id" as "ownershipId"
+
+  Scenario: Requested ownerships should show not up in saved searches
+    Given I am authorized as JWT provider user "invoerder_ownerships"
+    When I send a GET request to "/saved-searches/v3"
+    Then the response status should be "200"
+    And the JSON response should not include:
+    """
+    Aanbod %{name}
+    """
+
+  Scenario: Approved ownerships should show up in saved searches
+    Given I am authorized as JWT provider user "centraal_beheerder"
+    And I approve the ownership with ownershipId "%{ownershipId}"
+    And I am authorized as JWT provider user "invoerder_ownerships"
+    When I send a GET request to "/saved-searches/v3"
+    Then the response status should be "200"
+    And the JSON response at "/" should include "Aanbod %{name}"
+
+  Scenario: Rejected ownerships should show not up in saved searches
```

Comment:
Nice work on these extra acceptance tests, but they work on the same "ownershipId", so they can interfere. What will happen if you first run the reject scenario and then the approve scenario? I think that will fail because I a rejected ownership can not be approved. So better move the request for ownership inside the scenario instead of having it inside the background.

Reply:
They do not work on the same Ownership.
For each time the background is called a new organizer and a new ownership is created, so they should never clash.

Reply:
True! Discard that remark.